src/screens/GoalScreen/GoalScreen.tsx

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
-import { View } from 'react-native';
+import { View} from 'react-native';
 import withHeader from '../../helpers/withHeader';
 import ShortGoalsComponent from '../../components/ShortGoalsComponent/ShortGoalsComponent';
 const GoalScreen = () => {
   return (
-    <View>
+    <View style={styles.contain}>
       <ShortGoalsComponent />
```

Comment:
This should not change right?

Reply:
Sorry, forgot to remove that after testing my screen.